python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -866,6 +866,46 @@ def test_prune_previous_versions_with_write(arctic_library):
     assert v3.empty
 
 
+def test_prune_previous_versions_with_write_batch(library_factory):
```

Comment:
What does this test that isn't covered in `test_basic_version_store.py`?

Reply:
we also have tests for the same batch operations from both, library and store APIs

Reply:
We should try to only add similar tests of `NativeVersionStore` and `Library` if the behaviour is different. In this case it isn't, so remove this test

Reply:
agree